src/components/atoms/Button/Button.tsx

```diff
@@ -123,7 +122,9 @@ const SButton = styled.button<IButtonProps>`
     color: ${props => props.contrastColor};
     color: ${({ styling }) => styling && hoverFontColors[styling]};
```

Comment:
I know that this is not your code but it seems like a duplicate `color`?

Reply:
Yup I'll remove it 👌🏻